Makefile

```diff
@@ -1,16 +1,16 @@
 # Shortcuts for various tasks (UNIX only).
 # To use a specific Python version run:
-# $ make install PYTHON=python3.3
+# $ make install PYTHON=python3.7
 
 PYTHON = python3
```

Comment:
Should this be `PYTHON ?= python3` ?
https://www.gnu.org/software/make/manual/html_node/Setting.html

Reply:
agree, this should default to python3

Reply:
Linked doc says:

> If you’d like a variable to be set to a value only if it’s not already set, then you can use the shorthand operator ‘?=’ instead of ‘=’. 

PYTHON is already set, so there's no need to use `?=`

Reply:
Ahh, true. I just did a test with this `Makefile`:
```Makefile
PYTHON = python3
MYTHON ?= python3

.PHONY: test

test:
	@echo "PYTHON is set to $(PYTHON)"
	@echo "MYTHON is set to $(MYTHON)"
```
which gives these results:
```console
$ make test PYTHON=python3.8 MYTHON=python3.8
PYTHON is set to python3.8
MYTHON is set to python3.8
$ PYTHON=python3.8 MYTHON=python3.8 make test
PYTHON is set to python3
MYTHON is set to python3.8
```
so this was just a gap in my understanding of Makefile variables. Sorry!